src/extension/intellisenseHelper.ts

```diff
@@ -67,10 +67,12 @@ export class IntellisenseHelper {
         const typingsSource = path.resolve(__dirname, "..", "..", "ReactTypings");
         const reactTypings = path.resolve(typingsSource, "react");
         const reactNativeTypings = path.resolve(typingsSource, "react-native");
+        const typingsIndex = path.resolve(typingsSource, "react-native.d.ts");
 
         const typingsDestination = path.resolve(vscode.workspace.rootPath, ".vscode", "typings");
         const reactTypingsDestination = path.resolve(typingsDestination, "react");
         const reactNativeTypingsDestination = path.resolve(typingsDestination, "react-native");
+        const typingsIndexDestination = path.resolve(vscode.workspace.rootPath, "react-native.d.ts");
```

Comment:
Technically it's correct, though we can put this file into subfolder rather than to project root - tsserver understansds both locations.